apps/emqx_retainer/etc/emqx_retainer.conf

```diff
@@ -63,19 +63,16 @@ emqx_retainer: {
   ##
   ## Value: mnesia
   ##
-  connector:
-    [
-      {
-       type: mnesia
-       config: {
-            ## storage_type: ram | disc | disc_only
-            storage_type: ram
+  config: {
 
-            ## Maximum number of retained messages. 0 means no limit.
-            ##
-            ## Value: Number >= 0
-            max_retained_messages: 0
-       }
-      }
-    ]
+    type: mnesia
```

Comment:
I think this way might be better:
`store_to: built-in-database | mysql | postgresql`

Reply:
Taking into account that the retained message will be read from the database, `store_to` may not be appropriate, but `type` is too abstract, I think it can have a more appropriate name.